craft_parts/ctl.py

```diff
@@ -64,21 +65,20 @@ def _client(cmd: str, args: list[str]) -> str | None:
     :raise RuntimeError: If the command is invalid.
     """
     try:
-        call_fifo = os.environ["PARTS_CALL_FIFO"]
-        feedback_fifo = os.environ["PARTS_FEEDBACK_FIFO"]
+        ctl_socket_path = os.environ["PARTS_CTL_SOCKET"]
```

Comment:
what are these unrelated changes? Is `feature/offline-builds` up to date?

Reply:
It looks like they came from the merge I did. I've done a rebase and they disappeared. Not sure what happened.